docs/RELEASES.md

```diff
@@ -197,9 +205,17 @@ These are the general steps for a release:
 2. Draft release notes at [thunderbird-notes](https://github.com/thunderbird/thunderbird-notes).
 3. Trigger build via the [Shippable Build & Signing](https://github.com/thunderbird/thunderbird-android/actions/workflows/shippable_builds.yml) action.
 4. Review the build results by reviewing the action summary and the git commits resulting from the build.
-5. Smoke test the build.
+   - Make sure the version code is incremented properly and not wildly off
+   - Ensure the commits are correct
+   - Ensure the symlink `app-metadata` points to the right product at this commit
+5. Test the build in the internal testing track
+   - Release versions should be throuroughly tested with the test plan in Testrail
+   - Beta versions only require a basic smoke test to ensure it installs
 6. Promote TfA and K-9 releases in Play Store.
+   - Betas are only released for TfA. K-9 beta users are advised to use Thunderbird.
 7. Wait for Play Store review to complete.
-8. Update F-Droid to new TfA and K-9 releases.
-9. Send notification to Matrix channels, and beta or planning mailing lists.
+   - Release versions of TfA and K-9 have managed publishing enabled. Once the review has completed you need to publish the release
+   - Beta versions of TfA do not have managed publishing enabled. It will be available once Google has reviewed, even on a weekend.
+8. Update F-Droid to new TfA and K-9 releases by sending a pull request to [fdroiddata](https://gitlab.com/fdroid/fdroiddata)
+9. Send community updates to Matrix channels, and beta or planning mailing lists.
```

Comment:
Would you like an email sent for each release or would this be as-needed?

Reply:
I think as needed is fine for now. For the beta list we'd post if there are specific things we'd like to test on. For planning lists we could consider notifying about releases, but I want to be clear on the benefit.

Reply:
Added "as needed" for now